### PR TITLE
documentation(EJ2-60573): Data Annotation - Core and MVC.

### PR DESCRIPTION
**Pull request by sharmi70**

### CODE REVIEW

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/data-annotation.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+documentation: ug
+---
+
+# Data Annotation
+
+Data Annotations helps us to define rules to the model classes or properties to perform data validation and display suitable messages to end users.
+
+The Data Annotation can be enabled by setting the `TValue` property of Grid which maps the data annotations to the corresponding DataGrid Column property.
+
+> Grid Properties has more priority than Data Annotation. For Instance, if DisplayName Attribute is set to a Field in Grid Model class and also we set different value to the respective Grid Columns property HeaderText, then the value of HeaderText property will be considered and shown in Grid header.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/data-annotation.md:17]: >The Grid Properties has more priority than the Data Annotation. For Instance, if the DisplayName Attribute is set to a Field in the Grid Model class and set the different value to the respective Grid Columns property HeaderText, the value of the HeaderText property will be considered and shown in the Grid header.

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/data-annotation.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Data Annotation in ##Platform_Name## Grid Component
```
[the remark this answers is not in the document]
Comment (sharmi70) [reply ej2-asp-core-mvc/grid/EJ2_ASP.MVC/data-annotation.md:3]: this is not necessary to review.

[the remark this answers is not in the document]
Comment (Chrispineagunja) [reply ej2-asp-core-mvc/grid/EJ2_ASP.MVC/data-annotation.md:3]: I think it is necessary because the word **title** should start with a capital letter.

**Review by VigneshNatarajan27 — APPROVED** [review]
changes are fine.

File: ej2-asp-core-mvc/grid/data-annotation.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+description: Learn here all about Data Annotation in Syncfusion ##Platform_Name## Grid component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: Data Annotation
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# Data Annotation in Grid Component
+
+Data Annotations help you to define rules to the model classes or properties to perform data validation and display suitable messages to end users.
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/data-annotation.md:13]: In some places the Data Annotations is mentioned as plural and some places mentioned as singular like data annotation.  Maintain the consistency

**Review by MaithiliyK — COMMENTED** [review]
address my review comments

File: ej2-asp-core-mvc/grid/data-annotation.md (hunk 3 above)
Comment (sharmi70) [reply ej2-asp-core-mvc/grid/data-annotation.md]: mentioned changes completed.

**Pull request merged**